src/aat/resources/features/F-103/S-603.td.json

```diff
@@ -0,0 +1,36 @@
+{
+  "title": "must return 400 error response for mal-formatted user id list",
+  "_guid_": "S-603",
+  "productName": "CCD Data Store",
+  "operationName": "Get Case-Assigned Users and Roles",
+  "_extends_": "F-103_Test_Data_Base",
+
+
+"specs": [
+    "Richard - who can create a case",
+    "Jamal -  who is a privileged user with permissions to access the case assignments of other users",
+    "is made by Jamal with the Case ID of C1 & a malformed User ID list",
+    "has the 400 return code"
+  ],
+  "request": {
+    "pathVariables": {
+      "user_ids": "${[scenarioContext][testData][users][userDil][id]},,${[scenarioContext][testData][users][userRichard][id]}"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_400_Response",
+    "headers": {
+      "Content-Encoding": "gzip",
+      "Content-Length": "[[ANYTHING_PRESENT]]"
+    },
+    "body": {
+      "exception": "uk.gov.hmcts.ccd.endpoint.exceptions.BadRequestException",
+      "timestamp": "[[ANYTHING_PRESENT]]",
+      "message": "User ID is not valid",
+      "path": "[[ANYTHING_PRESENT]]",
+      "details": null,
+      "callbackErrors": null,
+      "callbackWarnings": null
+    }
```

Comment:
when you have a section of the test data inheriting from Common_XYZ_Response, you won't need to repeat many lines like these, as they will already be inherited from the base object

Reply:
Changed this, Pleaser review!!!